### ISSUE TITLE
Icinga2 crashes after a few seconds when starting dashing-icinga2

### ISSUE THREAD

**Issue by elpito** [mocdaniel/dashing-icinga2#68]
Hi,
After upgrading from Fedora 27 to 28 Icinga2 crashes, when I start dashing-icinga2. Because of distribution changes, I had to update the rubygem thin-1.6.4 to thin-1.7.2 (made the changes in Gemfile.lock). This is the error: `/usr/local/share/gems/gems/thin-1.6.4/lib/thin/server.rb:107: warning: constant ::Fixnum is deprecated`

The problem is reproducible, I just have to start dasing-icinga2.

This is the message in systemd, when icinga2 crashes:

```
icinga2[1392]: /usr/include/c++/8/bits/unique_ptr.h:328: typename std::add_lvalue_reference<_Tp>::type std::unique_ptr<_Tp, _Dp>::operator*() const [with _Tp = icinga::Expression; _Dp = std::default_delete<icinga::Expression>; typename std::add_lvalue_reference<_Tp>::type = icinga::Expression&]: Assertion 'get() != pointer()' failed.
```

Here the crashreport:
```
Application information:
  Application version: r2.9.1-1
  Installation root: /usr
  Sysconf directory: /etc
  Run directory: /run
  Local state directory: /var
  Package data directory: /usr/share/icinga2
  State path: /var/lib/icinga2/icinga2.state
  Modified attributes path: /var/lib/icinga2/modified-attributes.conf
  Objects path: /var/cache/icinga2/icinga2.debug
  Vars path: /var/cache/icinga2/icinga2.vars
  PID path: /run/icinga2/icinga2.pid

System information:
  Platform: Fedora
  Platform version: 28 (Server Edition)
  Kernel: Linux
  Kernel version: 4.17.11-100.fc27.x86_64
  Architecture: x86_64

Build information:
  Compiler: GNU 8.1.1
  Build host: unknown
Stacktrace:

	(0) libc.so.6: gsignal (+0x10b) [0x7fe4a6a7bfeb]
	(1) libc.so.6: abort (+0x12b) [0x7fe4a6a665c1]
	(2) /usr/lib64/icinga2/sbin/icinga2() [0x9db5d8]
	(3) icinga2: icinga::FilterUtility::GetFilterTargets(icinga::QueryDescription const&, boost::intrusive_ptr<icinga::Dictionary> const&, boost::intrusive_ptr<icinga::ApiUser> const&, icinga::String const&) (+0xd1c) [0x8c572c]
	(4) icinga2: icinga::ObjectQueryHandler::HandleRequest(boost::intrusive_ptr<icinga::ApiUser> const&, icinga::HttpRequest&, icinga::HttpResponse&, boost::intrusive_ptr<icinga::Dictionary> const&) (+0x5bc) [0x8c638c]
	(5) icinga2: icinga::HttpHandler::ProcessRequest(boost::intrusive_ptr<icinga::ApiUser> const&, icinga::HttpRequest&, icinga::HttpResponse&) (+0x4d7) [0x8c1c37]
	(6) icinga2: icinga::HttpServerConnection::ProcessMessageAsync(icinga::HttpRequest&, icinga::HttpResponse&, boost::intrusive_ptr<icinga::ApiUser> const&) (+0x3a) [0x8c1f5a]
	(7) icinga2: icinga::WorkQueue::RunTaskFunction(std::function<void ()> const&) (+0x32) [0x95a452]
	(8) icinga2: icinga::WorkQueue::WorkerThreadProc() (+0x965) [0x96a705]
	(9) libboost_thread.so.1.66.0: <unknown function> (+0x14af1) [0x7fe4a9530af1]
	(10) libpthread.so.0: <unknown function> (+0x7594) [0x7fe4a6e0b594]
	(11) libc.so.6: clone (+0x3f) [0x7fe4a6b3f0df]

***
* This would indicate a runtime problem or configuration error. If you believe this is a bug in Icinga 2
* please submit a bug report at https://github.com/Icinga/icinga2 and include this stack trace as well as any other
* information that might be useful in order to reproduce this problem.
***

[New LWP 8383]
[New LWP 8384]
[New LWP 8385]
[New LWP 8386]
[New LWP 8387]
[New LWP 8396]
[New LWP 8402]
[New LWP 8403]
[New LWP 8413]
[New LWP 8414]
[New LWP 8415]
[New LWP 8416]
[New LWP 8417]
[New LWP 8418]
[New LWP 8419]
[New LWP 8420]
[New LWP 8424]
[New LWP 8425]
[New LWP 8426]
[New LWP 8429]
[New LWP 8430]
[New LWP 8431]
[New LWP 8432]
[New LWP 8433]
[New LWP 8434]
[New LWP 8441]
[New LWP 8450]

warning: Loadable section ".note.gnu.property" outside of ELF segments

warning: Loadable section ".note.gnu.property" outside of ELF segments

warning: Loadable section ".note.gnu.property" outside of ELF segments
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/lib64/libthread_db.so.1".

warning: Loadable section ".note.gnu.property" outside of ELF segments

warning: Loadable section ".note.gnu.property" outside of ELF segments

warning: Loadable section ".note.gnu.property" outside of ELF segments

warning: Loadable section ".note.gnu.property" outside of ELF segments
0x00007fe4a6b0b9a0 in nanosleep () from /lib64/libc.so.6

Thread 28 (Thread 0x7fe4a070f700 (LWP 8450)):
#0  0x00007fe4a6e1592a in waitpid () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009c403e in icinga::Application::AttachDebugger(icinga::String const&, bool) ()
No symbol table info available.
#2  0x00000000009f48d2 in icinga::Application::SigAbrtHandler(int) ()
No symbol table info available.
#3  <signal handler called>
No symbol table info available.
#4  0x00007fe4a6a7bfeb in raise () from /lib64/libc.so.6
No symbol table info available.
#5  0x00007fe4a6a665c1 in abort () from /lib64/libc.so.6
No symbol table info available.
#6  0x00000000009db5d8 in std::__replacement_assert(char const*, int, char const*, char const*) ()
No symbol table info available.
#7  0x00000000008c572c in icinga::FilterUtility::GetFilterTargets(icinga::QueryDescription const&, boost::intrusive_ptr<icinga::Dictionary> const&, boost::intrusive_ptr<icinga::ApiUser> const&, icinga::String const&) ()
No symbol table info available.
#8  0x00000000008c638c in icinga::ObjectQueryHandler::HandleRequest(boost::intrusive_ptr<icinga::ApiUser> const&, icinga::HttpRequest&, icinga::HttpResponse&, boost::intrusive_ptr<icinga::Dictionary> const&) ()
No symbol table info available.
#9  0x00000000008c1c37 in icinga::HttpHandler::ProcessRequest(boost::intrusive_ptr<icinga::ApiUser> const&, icinga::HttpRequest&, icinga::HttpResponse&) ()
No symbol table info available.
#10 0x00000000008c1f5a in icinga::HttpServerConnection::ProcessMessageAsync(icinga::HttpRequest&, icinga::HttpResponse&, boost::intrusive_ptr<icinga::ApiUser> const&) ()
No symbol table info available.
#11 0x000000000095a452 in icinga::WorkQueue::RunTaskFunction(std::function<void ()> const&) ()
No symbol table info available.
#12 0x000000000096a705 in icinga::WorkQueue::WorkerThreadProc() ()
No symbol table info available.
#13 0x00007fe4a9530af1 in thread_proxy () from /lib64/libboost_thread.so.1.66.0
No symbol table info available.
#14 0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#15 0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 27 (Thread 0x7fe49bf3c700 (LWP 8441)):
#0  0x00007fe4a6e1152c in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e16ac in boost::condition_variable::wait(boost::unique_lock<boost::mutex>&) ()
No symbol table info available.
#2  0x000000000096a80e in icinga::WorkQueue::WorkerThreadProc() ()
No symbol table info available.
#3  0x00007fe4a9530af1 in thread_proxy () from /lib64/libboost_thread.so.1.66.0
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 26 (Thread 0x7fe4a060b700 (LWP 8434)):
#0  0x00007fe4a6e1152c in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e16ac in boost::condition_variable::wait(boost::unique_lock<boost::mutex>&) ()
No symbol table info available.
#2  0x000000000096a80e in icinga::WorkQueue::WorkerThreadProc() ()
No symbol table info available.
#3  0x00007fe4a9530af1 in thread_proxy () from /lib64/libboost_thread.so.1.66.0
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 25 (Thread 0x7fe49bcf3700 (LWP 8433)):
#0  0x00007fe4a6e1152c in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e16ac in boost::condition_variable::wait(boost::unique_lock<boost::mutex>&) ()
No symbol table info available.
#2  0x000000000096a80e in icinga::WorkQueue::WorkerThreadProc() ()
No symbol table info available.
#3  0x00007fe4a9530af1 in thread_proxy () from /lib64/libboost_thread.so.1.66.0
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 24 (Thread 0x7fe49bd34700 (LWP 8432)):
#0  0x00007fe4a6e1152c in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e16ac in boost::condition_variable::wait(boost::unique_lock<boost::mutex>&) ()
No symbol table info available.
#2  0x000000000096a80e in icinga::WorkQueue::WorkerThreadProc() ()
No symbol table info available.
#3  0x00007fe4a9530af1 in thread_proxy () from /lib64/libboost_thread.so.1.66.0
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 23 (Thread 0x7fe49bd75700 (LWP 8431)):
#0  0x00007fe4a6e1152c in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e16ac in boost::condition_variable::wait(boost::unique_lock<boost::mutex>&) ()
No symbol table info available.
#2  0x000000000096a80e in icinga::WorkQueue::WorkerThreadProc() ()
No symbol table info available.
#3  0x00007fe4a9530af1 in thread_proxy () from /lib64/libboost_thread.so.1.66.0
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 22 (Thread 0x7fe49bdb6700 (LWP 8430)):
#0  0x00007fe4a6e1152c in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e16ac in boost::condition_variable::wait(boost::unique_lock<boost::mutex>&) ()
No symbol table info available.
#2  0x000000000096a80e in icinga::WorkQueue::WorkerThreadProc() ()
No symbol table info available.
#3  0x00007fe4a9530af1 in thread_proxy () from /lib64/libboost_thread.so.1.66.0
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 21 (Thread 0x7fe49bdf7700 (LWP 8429)):
#0  0x00007fe4a6b34659 in poll () from /lib64/libc.so.6
No symbol table info available.
#1  0x00007fe4a082bb79 in pvio_socket_wait_io_or_timeout () from /lib64/libmariadb.so.3
No symbol table info available.
#2  0x00007fe4a082bf98 in pvio_socket_read () from /lib64/libmariadb.so.3
No symbol table info available.
#3  0x00007fe4a0837cb7 in ma_pvio_read () from /lib64/libmariadb.so.3
No symbol table info available.
#4  0x00007fe4a0837e95 in ma_pvio_cache_read () from /lib64/libmariadb.so.3
No symbol table info available.
#5  0x00007fe4a082e857 in ma_real_read () from /lib64/libmariadb.so.3
No symbol table info available.
#6  0x00007fe4a082f2cd in ma_net_read () from /lib64/libmariadb.so.3
No symbol table info available.
#7  0x00007fe4a0833fd1 in ma_net_safe_read () from /lib64/libmariadb.so.3
No symbol table info available.
#8  0x00007fe4a083701b in mthd_my_read_query_result () from /lib64/libmariadb.so.3
No symbol table info available.
#9  0x00000000006f4fc6 in icinga::IdoMysqlConnection::Query(icinga::String const&) ()
No symbol table info available.
#10 0x00000000006f8fae in icinga::IdoMysqlConnection::FinishConnect(double) ()
No symbol table info available.
#11 0x000000000095a452 in icinga::WorkQueue::RunTaskFunction(std::function<void ()> const&) ()
No symbol table info available.
#12 0x000000000096a705 in icinga::WorkQueue::WorkerThreadProc() ()
No symbol table info available.
#13 0x00007fe4a9530af1 in thread_proxy () from /lib64/libboost_thread.so.1.66.0
No symbol table info available.
#14 0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#15 0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 20 (Thread 0x7fe49be38700 (LWP 8426)):
#0  0x00007fe4a6b34659 in poll () from /lib64/libc.so.6
No symbol table info available.
#1  0x000000000098aa22 in icinga::Socket::Poll(bool, bool, timeval*) ()
No symbol table info available.
#2  0x00000000006be928 in icinga::LivestatusListener::ServerThreadProc() ()
No symbol table info available.
#3  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 19 (Thread 0x7fe49be79700 (LWP 8425)):
#0  0x00007fe4a6e1187a in pthread_cond_timedwait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e1aa4 in boost::condition_variable::do_wait_until(boost::unique_lock<boost::mutex>&, timespec const&) ()
No symbol table info available.
#2  0x0000000000767b46 in icinga::CheckerComponent::CheckThreadProc() ()
No symbol table info available.
#3  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 18 (Thread 0x7fe49beba700 (LWP 8424)):
#0  0x00007fe4a6b34659 in poll () from /lib64/libc.so.6
No symbol table info available.
#1  0x000000000098aa22 in icinga::Socket::Poll(bool, bool, timeval*) ()
No symbol table info available.
#2  0x000000000075b296 in icinga::ExternalCommandListener::CommandPipeThread(icinga::String const&) ()
No symbol table info available.
#3  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 17 (Thread 0x7fe49bf7d700 (LWP 8420)):
#0  0x00007fe4a6b3f3ff in epoll_wait () from /lib64/libc.so.6
No symbol table info available.
#1  0x0000000000990839 in icinga::SocketEventEngineEpoll::ThreadProc(int) ()
No symbol table info available.
#2  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#3  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#4  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 16 (Thread 0x7fe49bfbe700 (LWP 8419)):
#0  0x00007fe4a6b3f3ff in epoll_wait () from /lib64/libc.so.6
No symbol table info available.
#1  0x0000000000990839 in icinga::SocketEventEngineEpoll::ThreadProc(int) ()
No symbol table info available.
#2  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#3  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#4  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 15 (Thread 0x7fe49bfff700 (LWP 8418)):
#0  0x00007fe4a6b3f3ff in epoll_wait () from /lib64/libc.so.6
No symbol table info available.
#1  0x0000000000990839 in icinga::SocketEventEngineEpoll::ThreadProc(int) ()
No symbol table info available.
#2  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#3  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#4  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 14 (Thread 0x7fe4a0068700 (LWP 8417)):
#0  0x00007fe4a6b3f3ff in epoll_wait () from /lib64/libc.so.6
No symbol table info available.
#1  0x0000000000990839 in icinga::SocketEventEngineEpoll::ThreadProc(int) ()
No symbol table info available.
#2  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#3  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#4  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 13 (Thread 0x7fe4a00a9700 (LWP 8416)):
#0  0x00007fe4a6b3f3ff in epoll_wait () from /lib64/libc.so.6
No symbol table info available.
#1  0x0000000000990839 in icinga::SocketEventEngineEpoll::ThreadProc(int) ()
No symbol table info available.
#2  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#3  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#4  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 12 (Thread 0x7fe4a00ea700 (LWP 8415)):
#0  0x00007fe4a6b3f3ff in epoll_wait () from /lib64/libc.so.6
No symbol table info available.
#1  0x0000000000990839 in icinga::SocketEventEngineEpoll::ThreadProc(int) ()
No symbol table info available.
#2  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#3  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#4  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 11 (Thread 0x7fe4a0342700 (LWP 8414)):
#0  0x00007fe4a6b3f3ff in epoll_wait () from /lib64/libc.so.6
No symbol table info available.
#1  0x0000000000990839 in icinga::SocketEventEngineEpoll::ThreadProc(int) ()
No symbol table info available.
#2  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#3  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#4  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 10 (Thread 0x7fe4a0383700 (LWP 8413)):
#0  0x00007fe4a6b3f3ff in epoll_wait () from /lib64/libc.so.6
No symbol table info available.
#1  0x0000000000990839 in icinga::SocketEventEngineEpoll::ThreadProc(int) ()
No symbol table info available.
#2  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#3  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#4  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 9 (Thread 0x7fe4a0ca8700 (LWP 8403)):
#0  0x00007fe4a6e14dc7 in accept () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x000000000098ac3b in icinga::Socket::Accept() ()
No symbol table info available.
#2  0x00000000008e6f70 in icinga::ApiListener::ListenerThreadProc(boost::intrusive_ptr<icinga::Socket> const&) ()
No symbol table info available.
#3  0x000000000087650e in std::thread::_State_impl<std::thread::_Invoker<std::tuple<std::_Bind<void (icinga::ApiListener::*(icinga::ApiListener*, boost::intrusive_ptr<icinga::TcpSocket>))(boost::intrusive_ptr<icinga::Socket> const&)> > > >::_M_run() ()
No symbol table info available.
#4  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#5  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#6  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 8 (Thread 0x7fe4a99c4700 (LWP 8402)):
#0  0x00007fe4a6e1152c in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e16ac in boost::condition_variable::wait(boost::unique_lock<boost::mutex>&) ()
No symbol table info available.
#2  0x000000000096a80e in icinga::WorkQueue::WorkerThreadProc() ()
No symbol table info available.
#3  0x00007fe4a9530af1 in thread_proxy () from /lib64/libboost_thread.so.1.66.0
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 7 (Thread 0x7fe4a9a05700 (LWP 8396)):
#0  0x00007fe4a6e1187a in pthread_cond_timedwait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e1aa4 in boost::condition_variable::do_wait_until(boost::unique_lock<boost::mutex>&, timespec const&) ()
No symbol table info available.
#2  0x00000000009919cb in icinga::Timer::TimerThreadProc() ()
No symbol table info available.
#3  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 6 (Thread 0x7fe4a9a46700 (LWP 8387)):
#0  0x00007fe4a6e1187a in pthread_cond_timedwait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e1aa4 in boost::condition_variable::do_wait_until(boost::unique_lock<boost::mutex>&, timespec const&) ()
No symbol table info available.
#2  0x000000000098ef3b in icinga::ThreadPool::ManagerThreadProc() ()
No symbol table info available.
#3  0x00007fe4a768e523 in execute_native_thread_routine () from /lib64/libstdc++.so.6
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 5 (Thread 0x7fe4a9a87700 (LWP 8386)):
#0  0x00007fe4a6e1152c in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e16ac in boost::condition_variable::wait(boost::unique_lock<boost::mutex>&) ()
No symbol table info available.
#2  0x000000000098f87d in icinga::ThreadPool::WorkerThread::ThreadProc(icinga::ThreadPool::Queue&) ()
No symbol table info available.
#3  0x00007fe4a9530af1 in thread_proxy () from /lib64/libboost_thread.so.1.66.0
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 4 (Thread 0x7fe4a9b4a700 (LWP 8385)):
#0  0x00007fe4a6e1152c in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e16ac in boost::condition_variable::wait(boost::unique_lock<boost::mutex>&) ()
No symbol table info available.
#2  0x000000000098f87d in icinga::ThreadPool::WorkerThread::ThreadProc(icinga::ThreadPool::Queue&) ()
No symbol table info available.
#3  0x00007fe4a9530af1 in thread_proxy () from /lib64/libboost_thread.so.1.66.0
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 3 (Thread 0x7fe4a9b09700 (LWP 8384)):
#0  0x00007fe4a6e1152c in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e16ac in boost::condition_variable::wait(boost::unique_lock<boost::mutex>&) ()
No symbol table info available.
#2  0x000000000098f87d in icinga::ThreadPool::WorkerThread::ThreadProc(icinga::ThreadPool::Queue&) ()
No symbol table info available.
#3  0x00007fe4a9530af1 in thread_proxy () from /lib64/libboost_thread.so.1.66.0
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 2 (Thread 0x7fe4a9ac8700 (LWP 8383)):
#0  0x00007fe4a6e1152c in pthread_cond_wait@@GLIBC_2.3.2 () from /lib64/libpthread.so.0
No symbol table info available.
#1  0x00000000009e16ac in boost::condition_variable::wait(boost::unique_lock<boost::mutex>&) ()
No symbol table info available.
#2  0x000000000098f87d in icinga::ThreadPool::WorkerThread::ThreadProc(icinga::ThreadPool::Queue&) ()
No symbol table info available.
#3  0x00007fe4a9530af1 in thread_proxy () from /lib64/libboost_thread.so.1.66.0
No symbol table info available.
#4  0x00007fe4a6e0b594 in start_thread () from /lib64/libpthread.so.0
No symbol table info available.
#5  0x00007fe4a6b3f0df in clone () from /lib64/libc.so.6
No symbol table info available.

Thread 1 (Thread 0x7fe4a9b53900 (LWP 8359)):
#0  0x00007fe4a6b0b9a0 in nanosleep () from /lib64/libc.so.6
No symbol table info available.
#1  0x00007fe4a6b0b87a in sleep () from /lib64/libc.so.6
No symbol table info available.
#2  0x00000000009f54aa in icinga::Application::RunEventLoop() ()
No symbol table info available.
#3  0x00000000007f77fb in icinga::IcingaApplication::Main() ()
No symbol table info available.
#4  0x00000000009f2e1e in icinga::Application::Run() ()
No symbol table info available.
#5  0x000000000088acc5 in icinga::DaemonCommand::Run(boost::program_options::variables_map const&, std::vector<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> >, std::allocator<std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char> > > > const&) const ()
No symbol table info available.
#6  0x00000000009f782d in Main() ()
No symbol table info available.
#7  0x0000000000635ebd in main ()
No symbol table info available.
```

**Comment by dnsmichi** [comment]
Hi,

that's a problem with Icinga 2 then, nothing related to this project which just sources the API.

Cheers,
Michael

**Comment by elpito** [comment]
Thank you, should I place this in the Icinga2 project? I thought, because it's only happening, when enabling the dashboard, it should be placed here.

Cheers,
Daniel

**Comment by dnsmichi** [comment]
This has been reported already IIRC, I've fixed a similar thing for Icinga 2.9.2.
